internal/record/ticketsender.go

```diff
@@ -83,7 +83,7 @@ func (t *ticketSender) sendTicketsToS2A(sessionTickets [][]byte, callComplete ch
 					t.ensureProcessSessionTickets.Done()
 				}
 			}()
-			hsConn, err := service.Dial(t.hsAddr)
+			hsConn, err := service.Dial(context.Background(), t.hsAddr, nil)
```

Comment:
Please add a comment explaining why we're not doing anything interesting with the context or transport credentials here.

Reply:
done!